projects/commudle-admin/src/app/feature-modules/community-channels/components/channel-members/member/member.component.ts

```diff
@@ -1,11 +1,11 @@
-import { EUserRoles } from 'projects/shared-models/enums/user_roles.enum';
-import { Component, EventEmitter, Inject, Input, OnDestroy, OnInit, Output } from '@angular/core';
-import { EUserRolesUserStatus, IUserRolesUser } from 'projects/shared-models/user_roles_user.model';
-import { LibAuthwatchService } from 'projects/shared-services/lib-authwatch.service';
-import { NbMenuService, NbWindowService, NB_WINDOW } from '@nebular/theme';
-import { filter, map } from 'rxjs/operators';
-import { ICurrentUser } from 'projects/shared-models/current_user.model';
-import { UserChatComponent } from 'projects/shared-components/user-chat/user-chat.component';
+import {EUserRoles} from 'projects/shared-models/enums/user_roles.enum';
+import {Component, EventEmitter, Inject, Input, OnDestroy, OnInit, Output} from '@angular/core';
+import {EUserRolesUserStatus, IUserRolesUser} from 'projects/shared-models/user_roles_user.model';
+import {LibAuthwatchService} from 'projects/shared-services/lib-authwatch.service';
+import {NB_WINDOW, NbMenuService, NbWindowService} from '@nebular/theme';
+import {filter, map} from 'rxjs/operators';
+import {ICurrentUser} from 'projects/shared-models/current_user.model';
+import {UserChatsService} from '../../../../user-chats/services/user-chats.service';
```

Comment:
probably we can use absolute path from root, like projects/commudle-admin/......

Reply:
This is done.